add chromosomal instability methods (#350)
AlexsLemonade/OpenPBTA-manuscript: content/06.methods.md

For autosomes only, we set the status to "amplification" when the copy number value was greater than two times the ploidy value.
We plotted genome-wide gains and losses in (**Figure {@fig:S3}B**) using the R package `ComplexHeatmap` [@doi:10.1093/bioinformatics/btw313].

#### Breakpoint Density (`chromosomal-instability` analysis module)

We defined breakpoint density as the number of breaks per genome or exome per sample.
For Manta SV calls, we filtered to retain "PASS" variants and used breakpoints from the algorithm.

> For consensus CNV calls, if |log<sub>2</sub> ratio| > log<sub>2</sub>(1 + 0.5), we annotated the segment as a break. 

Reviewer: Can you point me to where you got `0.5` here please?

Author: Ah, I think I may have been looking at the wrong thing - the gap, but I see this default option for percent change is 0, so I will update this.

> For consensus CNV calls, if |log<sub>2<sub> ratio| > log<sub>2<sub>(1 + 0.5), we annotated the segment as a break. 

Reviewer: Perhaps this should be? I'm going to commit this to find out.
```suggestion
For consensus CNV calls, if |log<sub>2</sub> ratio| > log<sub>2</sub>(1 + 0.5), we annotated the segment as a break. 
```

Author: yes, that worked.

We then calculated breakpoint density as:

> $$\textrm{breakpoint density} = \frac{\textrm{N breaks x 1,000,000}}{\textrm{Size in bp of }\textit{effectively surveyed}\textrm{ genome or exome}}$$

Reviewer: If we want to make this consistent with how we talk about TMB, I think we could do the following:

```suggestion
$$\textrm{breakpoint density} = \frac{\textrm{N breaks}}{\textrm{Size in Mb of }\textit{effectively surveyed} \textrm{ genome}}$$
```

We would only have SV and CNV data on WGS samples, so we probably shouldn't mention exomes here.

Author: Ahh, I saw we did both, but you're right, correlations would only be on WGS in this case. For the bp-->Mb. I initially had Mb, but I checked the code and the size of the bed files is in bp, so I think it should be bp, and why we have to do bp/ 1M, which here I add at the top since I couldn't figure out how to make a fraction in a fraction in latex!

Reviewer: I did wonder why it was at the top, so that makes sense. The code and bed files for TMB are the same way: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/e03222f99311e0231c993fbc36fa344bdcb6b75a/analyses/snv-callers/util/tmb_functions.R#L142

Author: Oh I guess you are saying to just simplify this to not need to use 1M and instead just write Mb?

Reviewer: Yup.

Author: done

We consider tumor mutation burden (TMB) to be the number of consensus SNVs per effectively surveyed base of the genome.
We considered base pairs to be effectively surveyed if they were in the intersection of the genomic ranges considered by the callers used to generate the consensus and where appropriate, regions of interest, such as coding sequences.

> We calculated TMB as:

Reviewer: Let's see if adding a line break allows this to render in Markdown.
```suggestion
We calculated TMB as:

```
